packages/web-wallet/src/components/UnregisterTokenDialog.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import { X, Loader2, AlertCircle, CheckCircle } from 'lucide-react';
+
+interface UnregisterTokenDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onConfirm: () => Promise<void>;
+  tokenSymbol: string;
+  tokenName: string;
+}
+
+const UnregisterTokenDialog: React.FC<UnregisterTokenDialogProps> = ({
+  isOpen,
+  onClose,
+  onConfirm,
+  tokenSymbol,
+  tokenName,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isConfirmed, setIsConfirmed] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [success, setSuccess] = useState(false);
+
+  const handleUnregister = async () => {
```

Comment:
On the register token dialog we used `const { registerToken } = useWallet();` to register the token.
Why do we not have a similar method here for unregistering the token?

From the args it seems we receive the token and only notify the caller that the user intends to unregister and any actual action should be taken from the caller.